mail-client/mutt-wizard/mutt-wizard-0_pre20200430.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="A system for automatically configuring mutt and isync"
```

Comment:
This should say neomutt, too

Reply:
ooof, I keep forgetting this.  
thanks for noticing :smile_cat: